app/wama.web/Controllers/UserToolController.cs

```diff
@@ -239,7 +239,13 @@ public IActionResult AddCertification(string memberId)
             ViewBag.MemberId = memberId;
             return View($"{Constants.ADMIN_CONSOLE_USER_TOOL_DIRECTORY}/ViewCertification.cshtml", Certifications);
         }
-
+        public async Task<IActionResult> ViewOneCertification(string memberId) //need to straight idea first
```

Comment:
btw changing ViewOneCertification to ViewCertification could cause unwanted bug
since viewing multiple certification is named ViewCertification.
i dont think its good idea to change name of 2 class which one is already used and linked with views.